job_definitions/stats_snapshots.yml

```diff
@@ -11,7 +10,8 @@
 {#    ('g-cloud-16', '1fWI3LrnPU1cHVYo_83KWfgND-lng2ztM', False)    #}
 
 {% set frameworks = (
-    ('g-cloud-12', '1LVGqBCRhjpl2OrKaUouh8rEiV3dpBep_', True),
+    ('g-cloud-12', '1LVGqBCRhjpl2OrKaUouh8rEiV3dpBep_', False),
+    ('digital-outcomes-and-specialists-5', '1KqVLn0yHBt3Sd7BJrNwo1mw7C31HCpPB', True),
```

Comment:
Where does the `1KqVLn0yHBt3Sd7BJrNwo1mw7C31HCpPB` value come from out of interest?

Reply:
Oh are they Google Drive folder IDs or something?

Reply:
It's an existing Google Drive folder, a bunch of them were created a while back, by @benvand I think (see #300).